routes/cart.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+const router = express.Router();
+const mongoose = require('mongoose');
+const ObjectID = require('mongodb').ObjectID;
+
+router.post('/', async (req, res) => {
+  const db = mongoose.connection.db;
+  const cart = req.body.cart;
+  let data = [];
+  try {
+    for (const id of cart) {
+      let item = await db
+        .collection('menuItems')
+        .findOne({ _id: new ObjectID(id._id)})
```

Comment:
you can use `.find` and pass it an array to fetch all the items in one go instead of hitting the db for each item